docs/backend/installation/README.md

```diff
@@ -1,9 +1,3 @@
-# Badgeyay Backend
-
-> **Badgeyay provide an interface to event organizers for generating badges of events from concerts to conferences and meet-ups.**
```

Comment:
@abishekvashok Fix it

Reply:
Can't get your reviews. Please see carefully then criticize this is the main opening file in docs/backend/installation aka readme.md. what more than installation instructions should be present here? I can't see why this can't be approved. Did you ever see #2074

Reply:
Didnt see the short description point when ch you removed.